ansible/playbooks/roles/compatibility_layer/defaults/main.yml

```diff
@@ -17,8 +17,8 @@ gentoo_prefix_path: /cvmfs/{{ cvmfs_repository }}/versions/{{ eessi_version }}/c
 
 # How to build the prefix
 prefix_required_space: 15 GB
-prefix_snapshot_url: https://eessi-gentoo-snapshot.s3-eu-west-1.amazonaws.com
-prefix_snapshot_version: 20211120
+prefix_gentoo_overlay: https://github.com/gentoo/gentoo
+prefix_gentoo_commit: 7eaa2512d1e6ddb44e3b41bbddf6c74723f234ce
```

Comment:
Should we add a comment with some guidelines about how to pick a commit to use?

Reply:
Added a short comment in https://github.com/EESSI/compatibility-layer/pull/143/commits/e767711a2c881ae86f5970dc5d8b652be8b2d00e, I'd propose to maybe do that in more details in the (still to be written) documentation?